src/www/ui_components/server_card.ts

```diff
@@ -194,7 +184,7 @@ enum ServerCardState {
   @property({type: String}) serverAddress: string;
   @property({type: String}) serverId: string;
   @property({type: String}) serverName: string;
-  @property({type: String}) state: ServerCardState = ServerCardState.DISCONNECTED;
+  @property({type: String}) state: ServerConnectionState = ServerConnectionState.DISCONNECTED;
```

Comment:
This is effectively leaking implementation details, since it's a definition from the internal component. I think it's ok for now, since we already had this dependency before, but now it's more explicit.

Reply:
The other option is to map the connection enum to an animation enum (forward, reverse, off). It just felt a bit like faffing about, which is why I didn't do it. Would that work?

Reply:
Nah, let's not spend more time on this.